src/System.Composition/perftests/project.json

```diff
@@ -11,7 +11,7 @@
     "System.Runtime.Extensions": "4.0.10",
     "System.Threading": "4.0.10",
     "System.Threading.Tasks.Parallel": "4.0.0",
-    "System.Threading.Thread": "4.0.0-rc3-23910"
+    "System.Threading.Thread": "4.0.0-rc3-23918"
   },
   "frameworks": {
     "dnxcore50": {}
```

Comment:
I was hoping to see no more dnxcore50 but it does appear as though you left it in for the test projects so I guess we can clean that up another time.